tests/Fixer/Whitespace/BlankLineBeforeStatementFixerTest.php

```diff
@@ -1245,6 +1193,27 @@ public function testFixWithYield($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @dataProvider provideFixWithYieldCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithYieldFrom($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'statements' => ['yield_from'],
+        ]);
+
+        $expected = str_replace('yield', 'yield from', $expected);
```

Comment:
Replacing `yield` with `yield from` feels weird: `yield` expects any value (and possibly a key) while `yield from` only expects an `iterable`. Should't this test have its own cases?